PixelHunter1995/Player.cs

```diff
@@ -50,13 +50,11 @@ public Player(Game game, float x, float y)
             this.AnimationTileset = new AnimationTileset("Animations/felixia");
         }
 
-        public void Update(GameTime gameTime)
+        public void Update(GameTime gameTime, Input input)
         {
-            var mouseState = Mouse.GetState();
-
-            if (game.IsActive && mouseState.LeftButton == ButtonState.Pressed)
+            if (input.Actions.GetState(Inputs.Action.PLAYING_Move).IsDown)
             {
-                this.MovePosition = new Vector2(mouseState.X, mouseState.Y);
+                this.MovePosition = new Vector2(input.X, input.Y);
```

Comment:
I think we can afford to call it MouseX and MouseY.

Reply:
Sure, I used the same name as MouseState did, but I agree it is better to be explicit.